flows/vision-zero/make_kv_store.py

```diff
@@ -25,8 +25,23 @@
     "OCR_DIAGRAM_TARGET_PATH": os.getenv("OCR_DIAGRAM_TARGET_PATH"),
     "OCR_BATCH_SIZE": os.getenv("OCR_BATCH_SIZE"),
     "OCR_SINGLE_CRASH": os.getenv("OCR_SINGLE_CRASH"),
+    "AFD_DB_USERNAME": os.getenv("AFD_DB_USERNAME"),
+    "AFD_DB_PASSWORD": os.getenv("AFD_DB_PASSWORD"),
+    "AFD_DB_HOSTNAME": os.getenv("AFD_DB_HOSTNAME"),
+    "AFD_DB_PORT": os.getenv("AFD_DB_PORT"),
+    "AFD_DB_DATABASE": os.getenv("AFD_DB_DATABASE"),
+    "AWS_ACCESS_KEY_ID": os.getenv("AWS_ACCESS_KEY_ID"),
+    "AWS_SECRET_ACCESS_KEY": os.getenv("AWS_SECRET_ACCESS_KEY"),
+    "AFD_S3_SOURCE_BUCKET": os.getenv("AFD_S3_SOURCE_BUCKET"),
+    "AFD_S3_ARCHIVE_BUCKET": os.getenv("AFD_S3_ARCHIVE_BUCKET"),
+    "AFD_S3_SOURCE_PREFIX": os.getenv("AFD_S3_SOURCE_PREFIX"),
+    "AFD_S3_ARCHIVE_PREFIX": os.getenv("AFD_S3_ARCHIVE_PREFIX"),
+    "EMS_S3_SOURCE_BUCKET": os.getenv("EMS_S3_SOURCE_BUCKET"),
+    "EMS_S3_ARCHIVE_BUCKET": os.getenv("EMS_S3_ARCHIVE_BUCKET"),
+    "EMS_S3_SOURCE_PREFIX": os.getenv("EMS_S3_SOURCE_PREFIX"),
+    "EMS_S3_ARCHIVE_PREFIX": os.getenv("EMS_S3_ARCHIVE_PREFIX"),
 }
 
 json = json.dumps(kv_store)
 
-key_value_uuid = set_key_value(key="Vision Zero", value=json)
+key_value_uuid = set_key_value(key="Vision Zero Development", value=json)
```

Comment:
In our next prefect huddle, I think I want to propose that we reevaluate the benefit of a `production` branch. I am finding that I have a predictable set of micro-changes in the checkout that is in production, including things like setting this `key` parameter back to `Vision Zero`. Nothing urgent, but I'd like to get those floating changes into version control.

Reply:
I totally support of this. I think that I missed the initial discussions around this topic so I'd love to hear more. 🙏

Reply:
my memory is hazy, but I thought we had loosely agreed that a production branch didn't fit our needs. Are you proposing that we do need a production branch?

Reply:
@chiaberry - your memory is totally right, I have thought and expressed that we didn't need a `production` branch for the VZ deployments in the past. I'm not positive that I think we would benefit from one now, but I have encountered a situation where I have some files on the `atd-prefect` production checkout which are floating and I can't just commit them back into main. 

Everytime I look at this, I think I'm coming up with a new different opinion on how best to handle it, so really, I wanted to brainstorm with the group using this particular case as an example starting off point. 

It's also worth mentioning that the production `atd-prefect` checkout is complicated by needing to run the moped test environment maker which has not made it into `main` yet.